airbyte-integrations/connectors/source-zendesk-talk/unit_tests/unit_test.py

```diff
@@ -3,42 +3,5 @@
 #
 
 
-import pytest
```

Comment:
can we unit test all the local methods e.g: get state, pagination, params, etc..?